prow/cmd/webhook-server/helpers.go

```diff
@@ -268,9 +269,49 @@ func patchValidatingWebhookConfig(ctx context.Context, caPem string, client ctrl
 		return fmt.Errorf("failed to get validation webhook config: %w", err)
 	}
 	oldValidatingWebhook := validatingWebhookConfig.DeepCopy()
-	validatingWebhookConfig.Webhooks[0].ClientConfig.CABundle = caPemEncoded
+	validatingWebhookConfig.Webhooks[0].ClientConfig.CABundle = []byte(caPem)
 	if err := client.Patch(ctx, &validatingWebhookConfig, ctrlruntimeclient.MergeFrom(oldValidatingWebhook), patchOptions); err != nil {
 		return fmt.Errorf("failed to patch validation webhook config: %w", err)
 	}
 	return nil
 }
+
+func (wa *webhookAgent) fetchClusters(d time.Duration, ctx context.Context, statuses *map[string]plank.ClusterStatus, configAgent *config.Agent) (error) {
```

Comment:
IIUC This method runs on a goroutine that is continuously updating the webhookAgent.statuses. Please add a brief comment here